### PR TITLE
Graded Lab convert from java to mips

### PR DESCRIPTION
**Pull request by fsalzhrane**
Done.

### CODE REVIEW

File: src/App.asm [hunk 1]
```diff
@@ -1,6 +1,58 @@
...
+	li $v0, 5
+	syscall
+	move $t0, $v0                   #t0 is  count
+	
+	li $t1, 0 			                #t1 is  i
+	li $t2, -1                      #t2 is  top_student  
+
+	
+loop_begin:
+	blt $t1, $t0,loop_body
```
Comment (marwankau) [inline src/App.asm:26]: variables (`i` and `count`) used in while loop condition must be loaded from memory before such as:
```asm
...
lw  $t1, i
lw  $t0. count
blt $t1, t0, loop_body
...
```

File: src/App.asm [hunk 2]
```diff
@@ -1,6 +1,58 @@
 .data
 
+	      str1: .asciiz "Enter total number of students: "
+        str2: .asciiz "Enter student score: "
+        str3: .asciiz "Highest score is: "
+
```
Comment (marwankau) [inline src/App.asm:6]: all integer variables should be defined here, such as:
```asm
...
i: .word 0
score: .word 0
top_student: .word -1
count: .word 0
...
```

**Review by marwankau — COMMENTED** [review]
All variable should be defined and loaded from memory. In addition, each time when you want to use these variables, you load it from memory and when you change its value you need to store it to memory.

**Comment by marwankau** [comment]
good job

**Comment by fsalzhrane** [comment]
Thank you doc marawn , i was just scared of plagiarism so i tried to do the program in a different way i hope doc no less marks for that :) do you want me to edit the program?

**Pull request closed without merging**